docs/deprecated_runbooks/KubeVirtComponentExceedsRequestedCPU.md

```diff
@@ -0,0 +1,4 @@
+# KubeVirtComponentExceedsRequestedCPU [Deprecated]
+<!-- Edited by apinnick, Nov 2022-->
+
+This alert is deprecated and there is not an actual issue even if this alert fires, it can be ignored and silenced.
```

Comment:
@avlitman I propose to update to:
This alert has been deprecated; it does not indicate a genuine issue. If triggered, it may be safely ignored and silenced.

Reply:
Yes sounds better thank you!

Reply:
Fixed

Reply:
note that I also added [Deprecated] in the title